packages/c/cmark/xmake.lua

```diff
@@ -0,0 +1,21 @@
+package("cmark")
+    set_homepage("https://github.com/commonmark/cmark")
+    set_description("CommonMark parsing and rendering library and program in C")
+
+    add_urls("https://github.com/commonmark/cmark.git")
+    add_versions("0.31.0", "8fbf029685482827828b5858444157052f1b0a5f")
```

Comment:
Why not add [targz](https://github.com/commonmark/cmark/archive/refs/tags/0.31.0.tar.gz) archive?

Reply:
no particular reason, I can add it if you want

Reply:
We can use `xmake l scripts/new.lua github:commonmark/cmark` to automatically generate package, which will save some time

Reply:
oh I didn't know about that!